lstchain/reco/disp.py

```diff
@@ -114,6 +123,7 @@ def disp_parameters_event(hillas_parameters, source_pos_x, source_pos_y):
              hillas_parameters.y.to(u.m).value,
              source_pos_x.to(u.m).value,
              source_pos_y.to(u.m).value,
+             hillas_parameters.psi.to(u.rad).value
```

Comment:
why are we using the hillas parameters container only here? wouldn't it make sense to use it also on all the previous calls?

Reply:
`.to_value(u.rad)`

Reply:
(Also above)

Reply:
> why are we using the hillas parameters container only here? wouldn't it make sense to use it also on all the previous calls?

You mean passing the container instead of passing as argument the individual parameters, right?
I can agree, but that is not related to the big being fixed here, right?

Reply:
not related to the bug, just a suggestion

Reply:
Can we merge? I'd rather not change the arguments of the functions.